# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mnemoc/rsbac/rsbac-no_extraversion.diff
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	Fri Mar  7 16:24:43 2003
+++ ./Makefile	Fri Mar  7 16:25:10 2003
@@ -71,9 +71,9 @@
 #
 # Add RSBAC version
 #
-ifeq ($(CONFIG_RSBAC),y)
-EXTRAVERSION=-rsbac
-endif
+#ifeq ($(CONFIG_RSBAC),y)
+#EXTRAVERSION=-rsbac
+#endif
 
